packages/studio/tailwind.config.ts

```diff
@@ -50,6 +50,7 @@ export default {
   content: [
     path.resolve(__dirname, "src/**/*.{ts,tsx}"),
     path.resolve(__dirname, "index.html"),
+    path.join(path.dirname(require.resolve("@yext/studio-ui")), "**/*.js"),
```

Comment:
Is this intended to be `**/*.js`, or ts/tsx?

Reply:
js!